src/pages/blog/ce-qui-caractérise-le-top-20%-des-devs.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: ../../layouts/ContentLayout.astro
+---
+
+# Ce qui caractérise le top 20% des devs
+
+## Introduction
+
+Lorsque l'on parle des développeurs, il est fréquent de vouloir comprendre ce qui distingue les meilleurs, le fameux top 20%, du reste des développeurs.
```

Comment:
I would add the **why**: why do we want to understand the difference between the best developers and us?

Reply:
Done. 🙂